azurerm/resource_arm_databricks_workspace.go

```diff
@@ -0,0 +1,244 @@
+package azurerm
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/databricks/mgmt/2018-04-01/databricks"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmDatabricksWorkspace() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmDatabricksWorkspaceCreate,
+		Read:   resourceArmDatabricksWorkspaceRead,
+		Update: resourceArmDatabricksWorkspaceUpdate,
```

Comment:
from what I can see, we should be able to rename `Create` -> `CreateUpdate` and use the same method here, by making the `sku` ForceNew?